nebula_hw_interfaces/src/nebula_hesai_hw_interfaces/hesai_hw_interface.cpp

```diff
@@ -532,6 +542,32 @@ HesaiLidarRangeAll HesaiHwInterface::GetLidarRange()
   return hesai_range_all;
 }
 
+Status HesaiHwInterface::checkAndSetLidarRange(
+  const HesaiCalibrationConfigurationBase & calibration)
+{
+  if (sensor_configuration_->sensor_model == SensorModel::HESAI_PANDARAT128) {
+    return Status::SENSOR_CONFIG_ERROR;
+  }
+
+  int cloud_min = sensor_configuration_->cloud_min_angle * 10;
```

Comment:
Nit: Instead of multiplying by 10 and using 3600, it would be tidier to define decidegrees so it is clear when we are dealing in normal degrees or decidegrees.

Reply:
Agreed. Since a lot of the values come from other places that are in degrees, the `* 10` is almost inevitable.
The cleanest (but for this PR overkill) approach would be proper types like `Degrees`, `DeciDegrees` which can be cast properly. See [std::chrono::duration](https://en.cppreference.com/w/cpp/chrono/duration) for nice unit handling.